client/changes/VERSION_COMPAT

```diff
@@ -9,3 +9,5 @@
 # BEGIN DEPENDENCY LIST -------------------------
 # leap.foo.bar>=x.y.z
 pysqlcipher>2.6.3
+leap.common>=4.0
```

Comment:
confirmed, `leap.common` will be `0.4` on next release (note 0.4, not 4.0, typo fixed)